Update microsoft-365-monitoring.md (#12046)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/microsoft-365-monitoring.md

# Learn about Microsoft 365 monitoring

> You can use dashboards in the [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339) to monitor the health of various Microsoft services for your organization's Microsoft 365 subscription. This capability was initially started with Exchange Online and now getting expanded to other Microsoft services like Microsoft Teams, Microsoft 365 Apps and more service in future. 

Reviewer: ```suggestion
You can use dashboards in the [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339) to monitor the health of various Microsoft services for your organization's Microsoft 365 subscription. This capability initially started with Exchange Online and is now being expanded to other Microsoft services like Microsoft Teams, Microsoft 365 Apps, and more services in the future. 
```

Reviewer 2: @JohanFreelancer9 Please change the first part of the last sentence to "This capability initially started with Exchange Online and now is being expanded to other Microsoft services . . . "

Reviewer: Done. @ematayur, please commit these suggestions. Many thanks.

Author: thank you for the feedback. committed with the suggestions

> Microsoft 365 Monitoring enables increases observability and minimizes downtime through providing near real time, user usage telemetry data with enriched alerts in M365 Admin Center's Service Health dashboard. 

Reviewer: ```suggestion
Microsoft 365 monitoring increases observability and minimizes downtime by providing near real-time, user usage telemetry data with enriched alerts in Microsoft 365 admin center Service Health dashboard. 
```
